sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,118 @@
+# Mar'24 release notes
+Release notes for commit range [f4e0d3177338](https://github.com/intel/llvm/commit/f4ed132f243ab43816ebe826669d978139964df2).. [d2817d6d317db1](https://github.com/intel/llvm/commit/d2817d6d317db1143bb227168e85c409d5ab7c82)
+
+## New Features
+### SYCL Compiler
+
+- Added more available CPU for `-march` option in OpenCL AOT compiler. [7911773c]
+- Added support for additional AMD GPU targets. [c1ce15944]
+- Added C++11 ABI=0 support. [459e122a]
```

Comment:
This commit does not change anything in the compiler. Tagging @bso-intel here to comment about what's the best way to document this activity in release notes. Did we complete it in 2024.2? If not, should we say that our support is partial? Should we mention it at all? How should we name it? I don't think that `ABI=0` is meaningful, it is not a part of a C++ standard. If we want to mention specific gcc macro, we should mention it full name